doc/cabal-commands.rst

```diff
@@ -142,6 +182,58 @@ arguments and flags that are common to some or all of them.
     Already generated `build-info.json` files will be removed since they would be stale otherwise.
 
 
+cabal list-bin
+--------------
+
+``cabal list-bin`` will either; (a) display the path for a single exe or (b)
+complain that the target doesn't resolve to a single binary. It won't list
+multiple binary paths but knows which build products a package has.
+
+::
+
+    $ cabal list-bin cabal-install
+    cabal: The list-bin command is for finding a single binary at once. The target
+    'cabal-install' refers to the package cabal-install-3.7.0.0 which includes the
```

Comment:
Maybe I should drop the version suffix from `cabal-install-3.7.0.0`?

Reply:
You are showing real output, so I guess a suffix is fine. Perhaps you could fake it to 3.6.2 so that users don't wonder why they can't find this secrete paid version on Hackage?

Reply:
I dropped the version suffix to avoid the docs looking stale and out of date when we're at cabal-3.8.0.0 or later.

Reply:
I think inventing something (e.g. erasing some places) is worse than having an outdated version, following the principle of least surprise. Having older version in the example is less surprising to anyone reading this, I think.

But I don't feel strongly about it, it's just s nuisance. So fell free to leave it as is.

Reply:
@ulysses4ever I had tried cabal-install-_._.0.0 (as in don't care about version number) but that looked awkward.

Reply:
Seeing as `#` is commonly used to refer to number, we could go with `cabal-install-#.#.0.0` or `cabal-install-#.#.#.#`.

Reply:
Yep, `#` is probably a bit better.

Reply:
I put back the version suffix with hashes in place of the actual digits.